### ISSUE TITLE
How to add custom css class to table thead when using DataTables as a Service

### ISSUE THREAD

**Issue by mahbubahmed** [yajra/laravel-datatables#2706]
Could you please tell me how to add custom CSS class to table thead when using DataTables as a Service?

**Comment by yajra** [comment]
If you want to add class on the table: https://yajrabox.com/docs/laravel-datatables/master/html-builder-table

```php
{{ $dataTable->table(['class' => 'my-custom-class']) }}
```

For the TH and TD:

```php
Column::make('name')->addClass('custom-class'),
```

**Comment by joao-uefrom** [comment]
i have a custom style for my header so i would like to add the class just to thead, i think there is no way.

**Comment by joao-uefrom** [comment]
I added something like this to Builder.php and it worked as I would like.

    /**
     * @var string
     */
    protected $theadClass = '';

    public function theadClass($class)
    {
        $this->theadClass = $class;

        return $this;
    }

    /**
     * Generate DataTable's table html.
     */
    public function table(array $attributes = [], $drawFooter = false, $drawSearch = false)
    {
        ...
        $tableHtml .= '<thead class="' . $this->theadClass . '"><tr>' . implode('', $th) . '</tr>' . $searchHtml . '</thead>';
        ...
    }

**Comment by KevinEduardo** [comment]
How did you edited Builder? Did you copied somewhere? I'm a little lost here

**Comment by happymacarts** [comment]
@joao-uefrom  Upvote to add this to release!

**Comment by joao-uefrom** [comment]
> @joao-uefrom Upvote to add this to release!

I even made the pull request but as I don't know how to use git right, so I think it was not accepted. Thanks for trying for me :D
I didn't get to try again because I don't use the library anymore @-@.

**Comment by happymacarts** [comment]
I am pretty new to contributing to projects too, apparently I didn't do something right either, my PR was rejected too.

**Comment by yajra** [comment]
Sorry for the delays, but this feature is now implemented in v10.6. Thanks!
